lib/src/core/better_player_with_controls.dart

```diff
@@ -62,6 +63,19 @@ class _BetterPlayerWithControlsState extends State<BetterPlayerWithControls> {
     super.dispose();
   }
 
+  BetterPlayerAsmsTrack trackWithMaxBitrate() {
+    final asmsTracks = widget.controller!.betterPlayerAsmsTracks;
+    BetterPlayerAsmsTrack maxBitrateTrack =
+        BetterPlayerAsmsTrack.defaultTrack();
+
+    if (asmsTracks.length > 0 &&
+        widget.controller!.betterPlayerDataSource!.useInitialMaxBitrate!) {
+      maxBitrateTrack = asmsTracks.reduce(
+          (current, next) => current.bitrate! > next.bitrate! ? current : next);
```

Comment:
a skąd wiesz że `asmTracks.length >= 2` ? moze tu poleciec index out of bound exception

Reply:
Nie no tutaj jest że jak length jest > 0

Reply:
ok sorry, w dart to troche inaczej dziala - jeden element jak ejst to zwroci tylko go, bez odpalania tych warunkow https://api.dart.dev/stable/3.5.4/dart-core/Iterable/reduce.html

Reply:
a to tak, dlatego wziąłem tą funkcyjke bo miałem własną stworzyć ale poszukałem gotowej która jest zabezpieczona